homeassistant/components/frontend/www_static/home-assistant-polymer

```diff
@@ -1 +1 @@
-Subproject commit 75d760fc3e6f37485ae535a5476c8b00ecd2dfec
+Subproject commit db109f5dda043182a7e9647b161851e83be9b91e
```

Comment:
Checkout dev branch, pull from upstream, update submodule, checkout your feature branch, squash your commits, rebase off dev branch. That should do it.

The commit in the submodule you've changed is the latest in the dev branch, so you just need to make sure you are based off latest dev.